### PR TITLE
Closes #26 documentation add package authors

### LINKED ISSUE #26: Documentation:  Add package authors
### Please select a category the issue is focused on?

Other

### Let us know where something needs a refresh or put your idea here!

- @jwang-lilly as author/maintainer
- rest of @pharmaverse/admiralneuro in alphabetical order

Please provide your ORCID if you have one and would like it included.

### PR DESCRIPTION
**Pull request by Fanny-Gautier**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by Fanny-Gautier** [comment]
@pharmaverse/admiralneuro Please double-check that I didn't misspell your name or any company names, and that I didn't miss anyone.

@gg106046 Sorry, I don't know your full name, please let me know how I should update it. Thanks!

@ all, please feel free to take it from here, as I’ll be out of office for the next two weeks. Thanks.

File: DESCRIPTION [hunk 1]
```diff
@@ -2,16 +2,25 @@ Type: Package
 Package: admiralneuro
 Title: Neuroscience Extension Package for ADaM in 'R' Asset Library
 Version: 0.1.0
-Authors@R: 
-    person("Open", "Source", , "admiralneuro@pharmaverse.com", role = c("aut", "cre"))
+Authors@R: c(
+  person("Jian", "Wang", , "wang_jian_wj@lilly.com", role = c("aut", "cre")),
```
Comment (jwang-lilly) [inline DESCRIPTION:6]: Please add: comment = c(ORCID = "0009-0002-4677-3781"), thanks!

File: DESCRIPTION [hunk 2]
```diff
@@ -2,16 +2,27 @@ Type: Package
...
+  person("Chris", "Pelentrides", role = "aut"),
+  person("Edoardo", "Mancini", role = "aut"),
```
Comment (manciniedoardo) [inline DESCRIPTION:8]: Please add my orcid: 0009-0006-4899-8641

File: DESCRIPTION [hunk 3]
```diff
@@ -2,16 +2,27 @@ Type: Package
...
+  person("Eric", "Nantz", role = "aut"),
+  person("Fanny", "Gautier", role = "aut", comment = c(ORCID = "0009-0004-3581-0131")),
+  person("Gayatri", "G.", role = "aut"),
+  person("Leena", "Khatri", role = "aut"),
+  person("Meilin", "Jiang", role = "aut", comment = c(ORCID = "0000-0003-4515-4567")),
+  person("Miles", "Almond", role = "aut"),
+  person("Eli Lilly", role = c("cph", "fnd")),
+  person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
+  )
 Description: A toolbox for programming Clinical Data Standards Interchange
```
Comment (manciniedoardo) [inline DESCRIPTION:18]:
```suggestion
Description: A toolbox for programming neuroscience Clinical Data Standards Interchange
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]

**Review by meilinjiang — APPROVED** [review]

File: DESCRIPTION [hunk 4]
```diff
@@ -2,16 +2,27 @@ Type: Package
...
-Authors@R: 
-    person("Open", "Source", , "admiralneuro@pharmaverse.com", role = c("aut", "cre"))
+Authors@R: c(
+  person("Jian", "Wang", , "wang_jian_wj@lilly.com", role = c("aut", "cre")),
+  person("Chris", "Pelentrides", role = "aut"),
+  person("Edoardo", "Mancini", role = "aut"),
+  person("Eric", "Nantz", role = "aut"),
+  person("Fanny", "Gautier", role = "aut", comment = c(ORCID = "0009-0004-3581-0131")),
+  person("Gayatri", "G.", role = "aut"),
+  person("Leena", "Khatri", role = "aut"),
```
Comment (khatril) [inline DESCRIPTION:12]: please add my orcid: 0000-0002-2268-4023

File: DESCRIPTION [hunk 5]
```diff
@@ -2,16 +2,27 @@ Type: Package
...
+  person("Meilin", "Jiang", role = "aut", comment = c(ORCID = "0000-0003-4515-4567")),
+  person("Miles", "Almond", role = "aut"),
```
Comment (MilesAlmond) [inline DESCRIPTION:14]: Please add my ORCID: 0009-0007-1784-0355

**Review by manciniedoardo — APPROVED** [review]
Thanks @Fanny-Gautier !
